packages/errors/src/json-rpc-error.ts

```diff
@@ -93,7 +94,7 @@ export interface RpcSimulateTransactionResult {
 
 export function getSolanaErrorFromJsonRpcError({ code: rawCode, data, message }: RpcErrorResponse): SolanaError {
     let out: SolanaError;
-    const code = Number(rawCode);
+    const code = rawCode !== undefined ? Number(rawCode) : SOLANA_ERROR__UNRECOGNIZED_JSON_RPC_ERROR;
```

Comment:
In the raw chance should we check that NaN / are there possibly custom error codes an RPC could use that the client error map won't have?

Reply:
Yeah I guess if someone provides `{ code: "123" }` in the payload, this will be NaN so probably worth handling this case as a separate PR.